pages/pil/unscoped/courses/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = () => {
       .then(trainingCourse => {
         req.trainingCourseId = id;
         req.trainingCourse = trainingCourse;
-        req.model = trainingCourse;
+        req.model = res.locals.static.trainingCourse = trainingCourse;
```

Comment:
Prefer obvious assignment.

```suggestion
        req.model = trainingCourse;
        res.locals.static.trainingCourse = trainingCourse;
```